torch/lib/c10d/ProcessGroupNCCL.hpp

```diff
@@ -197,9 +198,7 @@ class ProcessGroupNCCL : public ProcessGroup {
   //
   // If created by WorkNCCL's getFuture API, FutureNCCL has a reference to
   // WorkNCCL's cudaEvents, NCCL collective's outputs, and the device index of
-  // outputs' device. Its value is NCCL collective's
-  // outputs. FutureNCCL only supports single-process single-device mode where
-  // the size of outputs is equal to 1.
+  // outputs' device. Its value is NCCL collective's outputs.
```

Comment:
device -> devices?

Reply:
Thanks, fixed (it'll be visible once I re-export it)